subprojects/docs/src/samples/userguide/ant/properties/kotlin/build.gradle.kts

```diff
@@ -0,0 +1,27 @@
+// tag::set-property[]
+ant.setProperty("buildDir", buildDir)
+ant.properties.set("buildDir", buildDir)
+ant.properties["buildDir"] = buildDir
+ant.withGroovyBuilder {
+    "property"("name" to "buildDir", "location" to "buildDir")
+}
```

Comment:
Is there a formatting preference for multi-line like this vs. maintaining a one-liner matching groovy?

Reply:
Matching Groovy unless the line gets longer than 80 chars I'd say.